### PR TITLE
Issue #302: add RightCurly and AvoidNestedBlocks's new suppressed violations

### PR DESCRIPTION
**Pull request by HuGanghui**
Issue #302: add RightCurly and AvoidNestedBlocks's new suppressed violations

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionjavapatchfilter/AvoidNestedBlocks/context/Test.java [hunk 1]
```diff
@@ -43,4 +43,10 @@ public void test() {
                 }
         }
     }
+
+    public void test2() {
+        {
```
Comment (baratali) [inline src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionjavapatchfilter/AvoidNestedBlocks/context/Test.java:48]: please add `// violation without filter`

File: src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionjavapatchfilter/RightCurly/context/Test.java [hunk 2]
```diff
@@ -28,5 +28,13 @@ else if (x < 0)
                 break;
             }
         }
+
+        try {
+
+        }
```
Comment (baratali) [inline src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionjavapatchfilter/RightCurly/context/Test.java:34]: please add `// violation without filter`

**Review by baratali — CHANGES_REQUESTED** [review]

File: src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionjavapatchfilter/AvoidNestedBlocks/context/Test.java (hunk 1 above)
Comment (HuGanghui) [reply src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionjavapatchfilter/AvoidNestedBlocks/context/Test.java:48]: done

File: src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionjavapatchfilter/RightCurly/context/Test.java (hunk 2 above)
Comment (HuGanghui) [reply src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionjavapatchfilter/RightCurly/context/Test.java]: done

**Review by baratali — APPROVED** [review]

**Pull request merged**